### PR TITLE
fix: links for testing.md and eth-tester in docs

### PR DESCRIPTION
**Pull request by 0xCookedRice**

### CODE REVIEW

File: docs/userguides/projects.md [hunk 1]
```diff
@@ -122,8 +122,8 @@ installed, giving you more flexibility in how you define your scripts.
...
 
 Use tests to verify your project. Testing is a complex topic, learn more about testing using Ape framework
-[here](docs/userguides/testing.md)
+[here](../docs/userguides/testing.md)
 
 You can test your project using the `ape test` command. The `ape test` command comes with the core-plugin `ape-test`.
 The `ape-test` plugin extends the popular python testing framework
-[pytest](https://docs.pytest.org/en/6.2.x/contents.html).
\ No newline at end of file
+[pytest](https://docs.pytest.org/en/6.2.x/contents.html).
```
Comment (fubuloubu) [inline docs/userguides/projects.md]: What was the change here?

File: docs/userguides/projects.md [hunk 2]
```diff
@@ -122,8 +122,8 @@ installed, giving you more flexibility in how you define your scripts.
 
 
 Use tests to verify your project. Testing is a complex topic, learn more about testing using Ape framework
-[here](docs/userguides/testing.md)
+[here](../docs/userguides/testing.md)
```
Comment (fubuloubu) [inline docs/userguides/projects.md:125]: Have to confirm docs still build correctly here

File: docs/userguides/projects.md (hunk 1 above)
Comment (0xCookedRice) [reply docs/userguides/projects.md]: i didn't change anything here, it just showed something changed automatically

File: docs/userguides/projects.md (hunk 2 above)
Comment (0xCookedRice) [reply docs/userguides/projects.md:125]: yea apologies, it should just be "./testing.md" instead of "docs/userguides/testing.md"

**Pull request closed without merging**